content/en/api/service_level_objectives/service_level_objectives_create.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Create a service level objective
+type: apicontent
+order: 29.01
+external_redirect: /api/#create-a-service-level-objective
+---
+
+## Create a service level objective
+
+If you manage and deploy SLOs programmatically, it's easier to define the SLO in the Datadog UI and [export its valid JSON][1].
+
+**ARGUMENTS**:
+
+* **`type`** [*required*]:
+    The type of the SLO, chosen from:
+
+| SLO Type     | type attribute value             |
```

Comment:
`type attribute value` 🤔 `event` type will eventually include other sources (apm, logs, ...) , while `monitor` aka a `time-based` SLO type probably won't for the near future. 

Maybe something like `Supported values` or something.